.gitignore

```diff
@@ -4,3 +4,6 @@ Gemfile.lock
 .bundle
 bundle
 coverage
+.idea/
```

Comment:
I would rather we not put a bunch of IDE specific ignores in the global .gitignore; you can use a global gitignore (https://help.github.com/articles/ignoring-files/) to ignore files specific to your IDE of choice, or have local ignores in `.git/info/exclude` to ignore files on a per-repository basis without modifying the project .gitignore.